core/src/main/scala/ai/h2o/sparkling/backend/utils/H2OContextExtensions.scala

```diff
@@ -144,37 +144,35 @@ trait H2OContextExtensions extends RestCommunication with RestApiUtils with Shel
     launchShellCommand(Seq[String]("yarn", "application", "-kill", yarnAppId.get))
   }
 
-  private def verifyWebOpen(nodes: Array[NodeDesc], conf: H2OConf): Unit = {
-    val nodesWithoutWeb = nodes.flatMap { node =>
-      try {
-        RestApiUtils.getCloudInfoFromNode(node, conf)
-        None
-      } catch {
-        case cause: RestApiException => Some((node, cause))
-      }
-    }
+  private def verifyWebOpen(conf: H2OConf): Unit = {
+    val endpoint = RestApiUtils.getClusterEndpoint(conf)
+    val verifyWebOpenV3 = RestApiUtils.query[VerifyWebOpenV3](endpoint, "/3/verifyWebOpen", conf)
+    val nodesWithoutWeb = verifyWebOpenV3.nodes_web_disabled
     if (nodesWithoutWeb.nonEmpty) {
-      throw new H2OClusterNotReachableException(
-        s"""
+      throw new H2OClusterNotReachableException(s"""
     The following worker nodes are not reachable, but belong to the cluster:
```

Comment:
Would't be possible to add information to the message that web endpoints of those nodes are disabled?